common/rfb/ServerCore.cxx

```diff
@@ -101,3 +101,21 @@ rfb::BoolParameter rfb::Server::queryConnect
 ("QueryConnect",
  "Prompt the local user to accept or reject incoming connections.",
  false);
+rfb::IntParameter rfb::Server::dynamicQualityMin
+("DynamicQualityMin",
+ "The minimum dynamic JPEG quality, 0 = low, 9 = high",
+ 7, 0, 9);
+rfb::IntParameter rfb::Server::dynamicQualityMax
+("DynamicQualityMax",
+ "The maximum dynamic JPEG quality, 0 = low, 9 = high",
+ 8, 0, 9);
+
+static void bandwidthPreset() {
+  rfb::Server::dynamicQualityMin.setParam(2);
+  rfb::Server::dynamicQualityMax.setParam(9);
+}
+
+rfb::PresetParameter rfb::Server::preferBandwidth
```

Comment:
I don't think we need a new parameter class for this. Just let it be a boolean and handle the logic in the encoder manager.